packages/validation/src/validators/open-policy-agent/rules.ts

```diff
@@ -89,6 +89,9 @@ export const DEFAULT_TRIVY_PLUGIN: PolicyMetadata = {
       help: {
         text: "Remove the SYS_ADMIN capability from 'containers[].securityContext.capabilities.add'.",
       },
+      defaultConfiguration: {
```

Comment:
From now on the problems in Monokle will standard show as a `warning`! I updated these few to showcase that the default rules can be overridden per case. I took all the once with critical security-severity (>7).